WordPress/Classes/Networking/Remote Objects/RemoteBlogSettings.h

```diff
@@ -7,6 +7,12 @@
 @property (copy) NSString *desc;
 @property (copy) NSNumber *privacy;
 
+// Reading
+@property (copy) NSNumber *relatedPostsAllowed;
```

Comment:
I think these should be strong, not copy, right? NSNumber instances are unique and not cached/reused like NSString.